generative_ai/function_calling/function_declaration_from_func.py

```diff
@@ -0,0 +1,74 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+from typing import List
+
+from vertexai.generative_models import FunctionDeclaration
+
+
+PROJECT_ID = os.getenv("GOOGLE_CLOUD_PROJECT")
+
+
+def function_declaration_from_func():
+    # [START generativeaionvertexai_gemini_function_calling_declare_from_function]
+    # Define a function. Could be a local function or you can import the requests library to call an API
+    def multiply_numbers(numbers: List[int]) -> int:
```

Comment:
This is _almost_ a copy of the [markdown sample](https://cloud.google.com/vertex-ai/generative-ai/docs/multimodal/function-calling#python-from-function). I added the typings to fix the API error.